packages/react/src/Heading/Heading.test.tsx

```diff
@@ -4,6 +4,14 @@ import { Heading } from './Heading'
 import '@testing-library/jest-dom'
 
 describe('Heading', () => {
+  it('renders a design system BEM class name', () => {
```

Comment:
We usually place this test after the 'it renders'-test. And `getByRole` is preferred to `querySelector`

Reply:
I’m aware, and was almost on it, but decided to prevent scope creep. It was just that while starting another ticket, I came across some tests using ‘large’ instead of ‘extra’ so I thought let’s just make them consistent. Then I replaced ‘additional’ with ‘extra’ in the test description to match, but that should be all, I think. I can create a ticket for these additional improvements.